### PR TITLE
MGDAPI-912 - rebase + fixes for 3scale-next-0.7.0

### PR DESCRIPTION
**Pull request by matskiv**
# Description
JIRA: https://issues.redhat.com/browse/MGDAPI-912
This PR contains a rebase and a few commits with fixes and manual updates.
Please review last few commits, that have been added by me.

Installation instruction for testing locally:
https://github.com/integr8ly/delorean/tree/master/docs/ews#installation

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # cloud.google.com/go v0.56.0
 cloud.google.com/go/compute/metadata
-# github.com/3scale/3scale-operator v0.2.1-0.20200730110533-c3b57b704d73
+# github.com/3scale/3scale-operator v0.2.1-0.20201120164439-360b1f382716
```
Comment (austincunningham) [inline vendor/modules.txt:3]: this text is only a comment but it looks to be pointing at an older version

Comment (austincunningham) [reply vendor/modules.txt:3]: https://github.com/3scale/3scale-operator/pull/532/commits sha looks to be 2.10

Comment (matskiv) [reply vendor/modules.txt:3]: This is correct. Commit sha is what matters here.
btw: no idea why its prefixed with "#" :)

**Comment by trepel** [comment]
I don't think I am able to review the code too thoroughly, but LGTM. I ran the operator from this branch locally (doing that EWS staff first) against the byoc OSD cluster and the 3scale 2.10 was indeed installed (seen in 3scale Admin Portal). The apicast-production DC indeed specified 250 MiB as memory limit.

**Comment by matskiv** [comment]
@trepel "rhoam-e2e" succeeded and "e2e" test only failed with flaky tests, so I think this could be merged now, would you mind adding `/lgtm` ?
P.S: I renamed the last commit and removed one debug line, so tests restarted, but since this is trivial change, we don't have to wait for them to finish.

**Pull request merged**